ci/build_container_centos/docker_push.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Can this script be refactored with the existing `docker_push.sh`?

Reply:
done